basic_nodes/src/main/java/club/mondaylunch/gatos/basicnodes/BooleanOperationNodeType.java

```diff
@@ -0,0 +1,59 @@
+package club.mondaylunch.gatos.basicnodes;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+import club.mondaylunch.gatos.core.data.DataBox;
+import club.mondaylunch.gatos.core.data.DataType;
+import club.mondaylunch.gatos.core.graph.connector.NodeConnector;
+import club.mondaylunch.gatos.core.graph.type.NodeType;
+
+public class BooleanOperationNodeType extends NodeType.Process {
+
+    @Override
+    public Map<String, DataBox<?>> settings() {
+        return Map.of(
+                "config", DataType.STRING.create("")
```

Comment:
IMO it'd be better to have the setting as an inner enum, & create a DataType of that enum as a static field in `BooleanOperationNodeType` (or in the enum), so that we always know that the value will be one of the provided ones. `VariableExtractionNodeType` in PR #42 does this.

Reply:
Also perhaps 'mode' would be a better name than 'config'